### DISCUSSION TITLE
The ControlNet module is not working?

### DISCUSSION THREAD

**Discussion by dkqjhoifh** [Acly/krita-ai-diffusion#873] [Q&A]
Pls what's going on and why can't the files that are required for these features be downloaded automatically? I have the XL version installed, do they only support SD1.5?
![QQ截图20240625222918](https://github.com/Acly/krita-ai-diffusion/assets/171809048/ffe78598-c37d-4fe5-a1d1-e825316399cb)

**Comment by Drakosha405** [comment]
if you have selected them in the list, they should load.  have you looked in the controlnet folder to see if these models are available
![image](https://github.com/Acly/krita-ai-diffusion/assets/146111805/b16f64cd-bc7d-4858-9fce-86569bad82d4)

**Comment by Acly** [comment]
I haven't listed Stencil for XL in downloads yet. You can find it here: https://huggingface.co/monster-labs/control_v1p_sdxl_qrcode_monster
but it's 5GB and when I tried it didn't work very well (the SD1.5 version is way better). So don't really recommend, but feel free to give it a try...

**Reply by dkqjhoifh** [reply]
How do I use the Face function in the Control Layer? I've searched all over your youtube tutorials, but there's no introduction to it, and after trying it myself, I found that it always generates some strange pictures, and I want to know the correct way to use it.

**Reply by EmPiFree** [reply]
Krita AI is using IP-Adapter for the Face function. You should take a into their repository: https://github.com/tencent-ailab/IP-Adapter
